platform-api/src/routers/forms.py

```
@@ -0,0 +1,115 @@
+    # Turnstile
+    if not await verify_turnstile(body.turnstile_token, client_ip, settings.optional_secrets.get("turnstile_secret_key")):
+        raise HTTPException(400, "Turnstile verification failed")
+
+    # Create Sanity draft
+    doc_id = await create_submission(body, settings, sanity)
+
+    # Discord notification (fire-and-forget)
+    try:
+        await notify_discord(body, settings)
+    except Exception:
+        pass  # Don't block submission on Discord failure
```

Ruff (0.15.11)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)